From a9d54b97a4f10b7a44e03da6361af932016d903b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Sun, 9 May 2021 23:43:21 +0200
Subject: [PATCH] arm64: dts: allwinner: Move nodes to avoid merge conflicts

Several of the branches merged into the integration branch add or
change nodes that happen to sit next to each other, so they conflict on
every rebase. Move them apart so each branch touches its own part of
the file. No functional change, node order does not matter here.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi  | 12 ++++-----
 .../dts/allwinner/sun50i-h6-orangepi-3.dts    |  8 +++---
 arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi  | 27 +++++++++----------
 3 files changed, 23 insertions(+), 24 deletions(-)

diff --git a/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi b/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
index 8726f71a1808..ebc64458084c 100644
--- a/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
+++ b/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
@@ -100,12 +100,6 @@ osc32k: osc32k-clk {
 		};
 	};
 
-	de: display-engine {
-		compatible = "allwinner,sun8i-h3-display-engine";
-		allwinner,pipelines = <&mixer0>;
-		status = "disabled";
-	};
-
 	sound_hdmi: sound_hdmi {
 		compatible = "allwinner,sun9i-a80-hdmi-audio",
 			     "allwinner,sun8i-h3-hdmi-audio";
@@ -120,6 +114,12 @@ cpu {
 		};
 	};
 
+	de: display-engine {
+		compatible = "allwinner,sun8i-h3-display-engine";
+		allwinner,pipelines = <&mixer0>;
+		status = "disabled";
+	};
+
 	scpi_protocol: scpi {
 		compatible = "arm,scpi";
 		mboxes = <&msgbox 2>, <&msgbox 3>;
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts
index dd7f81ef97b7..e05cf8eb5474 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3.dts
@@ -128,6 +128,10 @@ &gpu {
 	status = "okay";
 };
 
+&i2s1 {
+	status = "okay";
+};
+
 &hdmi {
 	status = "okay";
 };
@@ -156,10 +160,6 @@ &emac {
 	status = "okay";
 };
 
-&i2s1 {
-	status = "okay";
-};
-
 &mdio {
 	ext_rgmii_phy: ethernet-phy@1 {
 		compatible = "ethernet-phy-ieee802.3-c22";
diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
index 012275867108..00158f3bceb6 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
@@ -128,20 +128,6 @@ psci {
 		method = "smc";
 	};
 
-	sound_hdmi: sound_hdmi {
-		compatible = "allwinner,sun9i-a80-hdmi-audio",
-			     "allwinner,sun50i-h6-hdmi-audio";
-		status = "disabled";
-
-		codec {
-			sound-dai = <&hdmi>;
-		};
-
-		cpu {
-			sound-dai = <&i2s1>;
-		};
-	};
-
 	timer {
 		compatible = "arm,armv8-timer";
 		arm,no-tick-in-suspend;
@@ -155,6 +141,19 @@ timer {
 			(GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>;
 	};
 
+	sound_hdmi: sound_hdmi {
+		compatible = "allwinner,sun9i-a80-hdmi-audio",
+			     "allwinner,sun50i-h6-hdmi-audio";
+		status = "disabled";
+
+		codec {
+			sound-dai = <&hdmi>;
+		};
+
+		cpu {
+			sound-dai = <&i2s1>;
+		};
+	};
 
 	soc {
 		compatible = "simple-bus";
-- 
2.43.0

